addons/block_code/ui/picker/categories/category_factory.gd

```diff
@@ -404,6 +404,42 @@ static func get_general_blocks() -> Array[Block]:
 	b.category = "Math"
 	block_list.append(b)
 
+	b = BLOCKS["parameter_block"].instantiate()
+	b.block_name = "randf"
+	b.variant_type = TYPE_FLOAT
+	b.block_format = "randf"
+	b.statement = "randf()"
+	b.defaults = {}
+	b.category = "Math"
+	block_list.append(b)
+
+	b = BLOCKS["parameter_block"].instantiate()
+	b.block_name = "randf_range"
+	b.variant_type = TYPE_FLOAT
+	b.block_format = "randf_range({from: FLOAT}, {to: FLOAT})"
```

Comment:
This `block_format` goes to the UI label so you can be verbose: `Get random float from {from: FLOAT} to {to: FLOAT}`. And you can also add tooltip text with the explanation in Godot docs:
```
b.tooltip_text = "Returns a pseudo-random float between [i]from[/i] and [i]to[/i] (inclusive)."
```
The tooltip text supports rich text with [BBCode format.](https://docs.godotengine.org/en/stable/tutorials/ui/bbcode_in_richtextlabel.html)

Reply:
Will do, thank you for your suggestion

Reply:
Thanks to you for contributing this!